server/chat.ts

```diff
@@ -1208,7 +1208,8 @@ export class CommandContext extends MessageContext {
 	checkHTML(htmlContent: string | null) {
 		htmlContent = ('' + (htmlContent || '')).trim();
 		if (!htmlContent) return '';
-		if (/>here.?</i.test(htmlContent) || /click here/i.test(htmlContent)) {
+		const auth = this.room ? this.room.auth : Users.globalAuth;
+		if (auth.get(this.user.id) !== '*' && (/>here.?</i.test(htmlContent) || /click here/i.test(htmlContent))) {
```

Comment:
This would allow whitelisted users (which is just voice without the symbol) to use click here. Maybe just auth.get(...) === '*'?

Reply:
Are you misreading this?

Reply:
Yes, yes I am. I could have sworn it was a + earlier....